pkg/providers/vsphere/vsphere.go

```diff
@@ -1069,6 +1071,19 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, datacenterSpec v1alpha1.VSphe
 		"auditPolicy":                          common.GetAuditPolicy(),
 	}
 
+	if controlPlaneMachineSpec.Taints != nil {
+		var taints []string
+		for _, taint := range controlPlaneMachineSpec.Taints {
+			taintString := fmt.Sprintf("key: %s\n            value: %s\n            effect: %s",
```

Comment:
Why don't we do this looping/formatting in the template itself?
That way we don't have to do this weird spacing in code